math/src/elliptic_curve/short_weierstrass/curves/bls12_381/compression.rs

```diff
@@ -112,13 +104,13 @@ mod tests {
     fn test_zero_point() {
```

Comment:
Please rename this test

```suggestion
    fn test_point_not_in_subgroup() {
```